logging/google/cloud/logging/_helpers.py

```diff
@@ -21,7 +21,7 @@
 from google.cloud.logging.entries import StructEntry
 from google.cloud.logging.entries import TextEntry
 
-METADATA_URL = "http://metadata/computeMetadata/v1/"
+METADATA_URL = "http://metadata.google.internal/computeMetadata/v1/"
```

Comment:
If this is FQ now it may make sense for us to add a `.` so DNS doesn't try to further resolve.
```suggestion
METADATA_URL = "http://metadata.google.internal./computeMetadata/v1/"
```

Reply:
@crwilcox I'm not sure what the effect of this will be but as long as it still resolves in GCE it should be fine. Will give it a go at work tomorrow.

Reply:
Thanks @rsalmond. If that is the FQDN, the `.` will have no effect. That signals to DNS to not attempt to evaluate this as a subdomain but to respect this as a complete name already. It stops DNS from expanding the name across all possible search paths, which can save on lookups.